libraries-3/src/test/java/com/baeldung/jnats/NatsClientLiveTest.java

```diff
@@ -1,115 +1,124 @@
 package com.baeldung.jnats;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertNull;
+import io.nats.client.Message;
+import io.nats.client.Subscription;
+import org.junit.Test;
 
+import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.TimeUnit;
 
-import org.junit.Test;
-
-import io.nats.client.Message;
-import io.nats.client.SyncSubscription;
+import static org.junit.Assert.*;
 
 public class NatsClientLiveTest {
 
-    @Test
-    public void givenMessageExchange_MessagesReceived() throws Exception {
-
-        NatsClient client = connectClient();
-
-        SyncSubscription fooSubscription = client.subscribeSync("foo.bar");
-        SyncSubscription barSubscription = client.subscribeSync("bar.foo");
-        client.publishMessage("foo.bar", "bar.foo", "hello there");
-
-        Message message = fooSubscription.nextMessage(200);
-        assertNotNull("No message!", message);
-        assertEquals("hello there", new String(message.getData()));
-
-        client.publishMessage(message.getReplyTo(), message.getSubject(), "hello back");
-
-        message = barSubscription.nextMessage(200);
-        assertNotNull("No message!", message);
-        assertEquals("hello back", new String(message.getData()));
-    }
-
-
-    private NatsClient connectClient() {
+    private NatsClient connectClient() throws IOException, InterruptedException {
         return new NatsClient();
     }
 
     @Test
-    public void whenWildCardSubscription_andMatchTopic_MessageReceived() throws Exception {
-
-        NatsClient client = connectClient();
-
-        SyncSubscription fooSubscription = client.subscribeSync("foo.*");
-
-        client.publishMessage("foo.bar", "bar.foo", "hello there");
-
-        Message message = fooSubscription.nextMessage(200);
-        assertNotNull("No message!", message);
-        assertEquals("hello there", new String(message.getData()));
+    public void givenMessageExchangeViaPublishSubscribe_MessagesReceived() throws Exception {
+
```

Comment:
Remove the blank lines at the start of methods

Reply:
fixing